c7n/resources/pinpoint.py

```diff
@@ -0,0 +1,26 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+from c7n.manager import resources
+from c7n.query import QueryResourceManager, TypeInfo
+
+
+@resources.register('pinpoint-app')
+class PinpointApp(QueryResourceManager):
+    class resource_type(TypeInfo):
+        service = 'pinpoint'
+        arn_type = 'apps'
+        enum_spec = ('get_apps', 'ApplicationsResponse.Item', None)
+        name = "Name"
+        id = 'Id'
+        universal_taggable = True
+        cfn_type = 'AWS::Pinpoint::App'
+        arn = "Arn"
+        permission_prefix = 'mobiletargeting'
+
+    def augment(self, resources):
+        for resource in resources:
+            tags = resource.get('tags', {})
+            formatted_tags = [{'Key': k, 'Value': v} for k, v in tags.items()]
+            resource['Tags'] = formatted_tags
```

Comment:
This feels like a nice touch :+1: . As far as I can tell we don't _need_ it since [this](https://github.com/cloud-custodian/cloud-custodian/blob/2c8f8afb8d070f1f9df56d77ea69adfbd02e503a/c7n/filters/core.py#L247-L248) should allow the value filter's `tag:<tag>` convenience logic to still work. But having tags in both forms is probably handy.